src/main/java/edu/harvard/iq/dataverse/harvest/client/ClientHarvestRun.java

```diff
@@ -40,12 +40,13 @@ public void setId(Long id) {
         this.id = id;
     }
 
-    public enum RunResultType { SUCCESS, FAILURE, INPROGRESS };
+    public enum RunResultType { SUCCESS, RUN_FAILED, RUN_IN_PROGRESS, DELETE_FAILED };
```

Comment:
Not a big deal, but what is the value of adding this status condition for a failure to delete a client? It has some value for a dataverse admin, to know that the last attempt to harvest from a certain server resulted in a failure. Is it really useful to know that an attempt to delete it failed? - should the startup check simply remove the "delete in progress" flag quietly instead?
After all, seeing how the client is still there makes it somewhat clear that the attempt to get rid of it didn't work out, so they should try again?

I may be missing some situation where it could actually be useful - so I'm open to hearing it.

Reply:
@landreev I guess handling it quietly would be ok. So instead of DELETE FAILED or FAILED we just show nothing?